test/server.test.ts

```diff
@@ -191,7 +216,8 @@ describe("Server", () => {
       return request(server.expressApp).get("/foo").expect(200, "foo");
     });
 
-    it("allows you to overwrite the root path", () => {
+    // webhooks overwrite the root path
+    xit("allows you to overwrite the root path", () => {
```

Comment:
this basically doesnt work anymore, because @octokit/webhooks createNodeMiddleware already sets root path "/" so overwriting is not possible.

Reply:
That doesn't seem right, as said in another comment, it only sets a default path

Reply:
done